control/bgpgracefulrestart.yaml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.3
+info:
+  title: BGP Graceful Restart Control models
+  description: >-
+    BGP Graceful Restart Control schemas
+  version: ^0.0.0
+components:
+  schemas:  
+    Bgp.GraceFulRestart:
+      description: >-
+        Initiates BGP Graceful Restart process for the selected BGP peers.
+      type: object
+      properties:
+        peer_names:
+          description: >-
+            The names of device BGP peer objects to control.
+            If no names are specified then all BGP peer objects that match the
+            x-constraint will be affected.
```

Comment:
x-constraint is applicable even when specific names are specified, and implicit for the peers in this context. So, can we remove mentioning x-constraint only for no-names?